vaadin-grid-flow-parent/vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/Grid.java

```diff
@@ -135,7 +135,7 @@
  *
  */
 @Tag("vaadin-grid")
-@NpmPackage(value = "@vaadin/vaadin-grid", version = "5.7.1")
+@NpmPackage(value = "@vaadin/vaadin-grid", version = "5.7.0")
```

Comment:
Is this right?

Reply:
It's currently in origin master https://github.com/vaadin/vaadin-grid-flow/blob/master/vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/Grid.java#L138